### PR TITLE
Functionality to add seasons using form.

### PR DESCRIPTION
**Pull request by rcoutin**
Seasons are still in memory. Meaning if you reload the app the data is lost.

+ button navigates to the form where you can type in "2019/20" for example and it will be added to the set and displayed on homepage.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,111 +1,139 @@
...
-  // how it looks.
-
-  // This class is the configuration for the state. It holds the values (in this
-  // case the title) provided by the parent (in this case the App widget) and
-  // used by the build method of the State. Fields in a Widget subclass are
-  // always marked "final".
-
-  final String title;
+class StartAppState extends State<StartApp> {
+  List<Season> _seasons = <Season>[];
```
Comment (malvika0901) [inline lib/main.dart]: do we have to use underscore in variable names?

File: lib/main.dart [hunk 2]
```diff
@@ -1,111 +1,139 @@
...
+                  ),
+                ),
+              ],
+            ),
+          ),
+        );
+      },
+    ));
+  }
+}
```
Comment (malvika0901) [inline lib/main.dart]: probably try to get rid of this long parentheses pyramid?

File: lib/main.dart [hunk 3]
```diff
@@ -1,111 +1,139 @@
+import 'dart:ui' as prefix0;
+
 import 'package:flutter/material.dart';
+import 'package:english_words/english_words.dart';
```
Comment (malvika0901) [inline lib/main.dart]: question : what is this for?

Comment (rcoutin) [reply lib/main.dart]: Was from earlier tutorial. Removing.

File: lib/main.dart (hunk 1 above)
Comment (rcoutin) [reply lib/main.dart]: No. I thought that was the convention. Changing.

File: lib/main.dart (hunk 2 above)
Comment (rcoutin) [reply lib/main.dart]: I disagree. If you see the code above, every parameter needs to be in line for readability. Moreover, Android Studio helps us in understanding what closing bracket is for what as in this pic. 
image

Getting all of them in one line would be more confusing IMO.

Comment (malvika0901) [reply lib/main.dart]: Okay. Makes sense

**Pull request merged**